### PR TITLE
simple-phpunit requires the zip extension

### PR DESCRIPTION
**Pull request by dbu**
without the zip extension enabled, i get `PHP Fatal error:  Uncaught Error: Class 'ZipArchive' not found in .../vendor/bin/simple-phpunit:46`

| Q             | A
| ------------- | ---
| Branch?       | 3.2 (first version containing the script)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #21060
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
:+1:

File: src/Symfony/Bridge/PhpUnit/bin/simple-phpunit [hunk 1]
```diff
@@ -43,6 +43,9 @@ if (!file_exists("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit") || md5_file(__
         @unlink("$PHPUNIT_VERSION.zip");
         passthru("wget https://github.com/sebastianbergmann/phpunit/archive/$PHPUNIT_VERSION.zip");
     }
+    if (!class_exists('ZipArchive')) {
+        throw new \Exception('simple-phpunit requires the PHP extension for zip support to be installed and enabled in order to download the right version of phpunit');
```
Comment (javiereguiluz) [inline src/Symfony/Bridge/PhpUnit/bin/simple-phpunit]: Proposal for a slightly different error message:

```
simple-phpunit requires the "zip" PHP extension to be installed and enabled in order to uncompress the downloaded PHPUnit packages
```

Comment (dbu) [inline src/Symfony/Bridge/PhpUnit/bin/simple-phpunit]: yep, thats better. updated

File: src/Symfony/Bridge/PhpUnit/bin/simple-phpunit [hunk 2]
```diff
@@ -43,6 +43,9 @@ if (!file_exists("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit") || md5_file(__
         @unlink("$PHPUNIT_VERSION.zip");
         passthru("wget https://github.com/sebastianbergmann/phpunit/archive/$PHPUNIT_VERSION.zip");
     }
+    if (!class_exists('ZipArchive')) {
+        throw new \Exception('simple-phpunit requires the "zip" PHP extension to be installed and enabled in order to uncompress the downloaded PHPUnit packages');
```
Comment (xabbuh) [inline src/Symfony/Bridge/PhpUnit/bin/simple-phpunit]: We usually terminate exception messages with a dot.

**Comment by xabbuh** [comment]
👍

File: src/Symfony/Bridge/PhpUnit/bin/simple-phpunit (hunk 2 above)
Comment (dbu) [inline src/Symfony/Bridge/PhpUnit/bin/simple-phpunit]: makes sense, fixed.

**Comment by dunglas** [comment]
👍

**Comment by javiereguiluz** [comment]
👍

**Comment by stof** [comment]
Thank you @dbu.

**Pull request merged**